blockchain/state_manager/src/vm_circ_supply.rs

```diff
@@ -203,9 +173,7 @@ fn get_fil_vested(genesis_info: &GenesisInfo, height: ChainEpoch) -> TokenAmount
     return_value
 }
 
-fn get_fil_mined<DB: BlockStore>(
-    state_tree: &StateTree<DB>,
-) -> Result<TokenAmount, Box<dyn StdError>> {
+fn get_fil_mined<DB: BlockStore>(state_tree: &StateTree<DB>) -> Result<TokenAmount, anyhow::Error> {
     let actor = state_tree
         .get_actor(reward::ADDRESS)?
         .ok_or_else(|| Error::State("Power actor address could not be resolved".to_string()))?;
```

Comment:
Not related to your PR but it's interesting that the error message is talking about the power actor address but it's actually trying to resolve the reward address.